ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml

```diff
@@ -29,34 +29,39 @@ tests:
       BASE_DOMAIN: perfscale.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "120"
       COMPUTE_NODE_TYPE: m5.xlarge
-      SIZE_VARIANT: large
+      CONTROL_PLANE_INSTANCE_TYPE: m5.4xlarge
     post:
     - chain: ipi-aws-post
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
     workflow: openshift-qe-control-plane
-  timeout: 5h0m0s
+  timeout: 6h0m0s
```

Comment:
Shall we do the same change for 4.15 release? 4.13 and 4.14 are all 6h now, 5.15 is still 5h.

Reply:
Ack, will update the timeout in 4.15 as well!